overcloud.yml

```diff
@@ -265,12 +265,16 @@
             src: osd-spec.yml.j2
             dest: "{{ osd_spec_file }}"
             force: yes
+
+        - name: set facts for ceph deployment
+          set_fact:
+            ceph_params: "{{ ('--storage-backend ceph --storage-external no --ceph-osd-spec-file ' +  osd_spec_file) }}"
```

Comment:
ceph_params: "--storage-backend ceph --storage-external no --ceph-osd-spec-file {{ osd_spec_file }}"
can you check this will work?

Reply:
Yes